src/App.tsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+// export function
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+    const searchQuery = query.trim().toLowerCase();
+
+    return (title.toLowerCase().includes(searchQuery)
```

Comment:
it would be nice to create separate function for lowerCasing and `includes` check